src/main/java/core/selenium/WebDriverManager.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package core.selenium;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.concurrent.TimeUnit;
+
+public final class WebDriverManager {
+
+    private WebDriverConfig webDriverConfig = WebDriverConfig.getInstance();
+    private WebDriver driver;
+    private WebDriverWait webDriverWait;
+    private ChromeOptions chromeOptions;
+    private static WebDriverManager instance = null;
+
+    /**
+     * Constructor of page WebDriverManager.
+     */
+    private WebDriverManager() {
+        initialize();
+    }
+
+    /**
+     * Gets Instance of WebElement..
+     *
+     * @return Instance of WebElement.
+     */
+    public static WebDriverManager getInstance() {
+        if (instance == null) {
+            instance = new WebDriverManager();
+        }
+        return instance;
+    }
+
+    /**
+     * Initializes According the config file.
+     */
+    private void initialize() {
+        io.github.bonigarcia.wdm.WebDriverManager.chromedriver().setup();
```

Comment:
This solution only works for chromedriver. You should refactor it to allow more types of drivers.